io/disk/vfc-tests/virtual_fc.py

```diff
@@ -351,7 +351,7 @@ def is_path_online():
                                    " % (operation, self.path, path_stat))
                     return True
             else:
-                self.log.info("Operation unknown, provide correct opertion")
+                self.log.info("Operation unknown, provide correct operation")
```

Comment:
@abdhaleegit these are intentionally made once, so that line length is <79 character.

Reply:
any way Thanks for changing them to proper spells.

Reply:
need the pep8 changes here too,
```

[root@server ~]# pycodestyle vfc.py 
vfc.py:286:80: E501 line too long (80 > 79 characters)
vfc.py:301:80: E501 line too long (80 > 79 characters)
[root@server ~]#
```

Reply:
if lines go beyond 79 chars.. than better to make it in 2 lines, instead triming words :-)

Reply:
done